stages/org.osbuild.grub2.iso.legacy

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+import os
+import shutil
+import string
+import sys
+
+import osbuild.api
+
+# The main grub2 configuration file template. Used for BIOS.
```

Comment:
Maybe a small comment here (and maybe also in org.osbuild.grub.iso) that we should keep the two templates in sync (assuming we want this)

Reply:
Yeah, and I also could forsee a future where the template is passed to it instead of embedded in it but that can wait.